RecordBuilders/Providers.php

```diff
@@ -50,7 +50,7 @@ protected function aggregate(ArchiveProcessor $archiveProcessor): array
                 Metrics::INDEX_NB_VISITS_CONVERTED => $row[Metrics::INDEX_NB_VISITS_CONVERTED],
             ];
 
-            $record->sumRowWithLabel($row['label'], $columns);
+            $record->sumRowWithLabel($row['label'] ?: '', $columns);
```

Comment:
Strictly speaking this guards against all falsey values, not just `null` as per the PR title. Should we allow falsey labels such as `'0'` and change to `??` insted of `?:`?

Reply:
Given a provider will never be named `'0'` I think it's fine to leave it. But I don't see a problem w/ changing it either.

Reply:
What's there should work and there's no need to change it, but `??` would be a little safer as it would account for the key or even array not existing.